docs/development_environment.mdx

```diff
@@ -42,7 +42,7 @@ In the future, if you want to get back to your development environment: open Vis
 
 The devcontainer comes with some useful tasks to help you with development, you can start these tasks by opening the command palette and select `Tasks: Run Task` then select the task you want to run.
 
-When a task is currently running (like `Preview` for the docs), it can be restarted by opening the command palette and selecting `Tasks: Restart Running Task`, then select the task you want to restart.
+When a task is currently running (like `Preview` for the docs), it can be restarted by opening the command palette with `Ctrl+Shift+P` and selecting `Tasks: Restart Running Task`, then select the task you want to restart.
```

Comment:
```suggestion
When a task is currently running (like `Preview` for the docs), it can be restarted by opening the command palette with `Shift`+`Command`+`P`(Mac) / `Ctrl`+`Shift`+`P` (Windows/Linux) and selecting `Tasks: Restart Running Task`, then select the task you want to restart.
```